src/cli/utils.js

```diff
@@ -78,6 +78,41 @@ export function error(...msgs) {
   console.error('  ', emoji.no, colors.error(msgs.join(' ')))
 }
 
+/**
+ * Clears the console.
+ */
+export function clear() {
+  console.clear()
+}
+
+/**
+ * Logs to the console with the last item pulled to the right edge of the terminal window.
+ *
+ * @param {...(string | [string, string])} [items] - When an item is a tuple of two strings, the first value is a
+ * modifier (e.g. color) that does not take any vertical space on the screen (i.e. its characters are not occupying
+ * available columns). The second value is a string that is visible to the user and does take vertical space.
+ */
+export function pad(...items) {
+  const paddingWidth =
+    process.stdout.columns -
+    // all strings
+    sum(items.map(item => (isArray(item) ? item[1] : item).length)) -
+    // all spaces between strings
+    (items.length - 1) -
+    // additional space character from the padding section
+    1
+
+  const padding = Array(paddingWidth || 0)
+    .fill(' ')
+    .join('')
+
+  log(
+    ...initial(items).map(item => (isArray(item) ? item[0](item[1]) : item)),
+    padding,
+    (item => (isArray(item) ? item[0](item[1]) : item))(last(items))
+  )
+}
```

Comment:
This function looks a little fragile to me, but I don't know how to make it better. Is there a package to easily offset things to the right edge? Looking forward to your feedback 🤗